.gitignore

```diff
@@ -9,6 +9,7 @@ out
 *.log*
 .env
 .vite
+yarn.lock
```

Comment:
This shouldn't be done 
https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control